.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "cortex-debug.armToolchainPath": "/usr/local/bin/",
+  "cortex-debug.openocdPath": "/usr/local/bin/openocd",
```

Comment:
Same issues, this is not going to be usable outside of mac os.

We've been talking about it a lot and decided not to add vscode solution if it is not working on all platforms.

Reply:
Before it supports Linux and Windows it could be named Upload & Run Flipper (macOS), or may be implemented only for macOS, VSCode launch-files allow that. And to show something like 'not implemented yet' on other platforms.

Having ready shortcut for debugging in macOS is much better than not having any shortcuts at all. It's not easy to configure VSCode to debug this.